tools/internal_ci/linux/psm-security.cfg

```diff
@@ -15,7 +15,7 @@
 # Config file for the internal CI (in protobuf text format)
 
 # Location of the continuous shell script in repository.
-build_file: "grpc/tools/internal_ci/linux/grpc_xds_k8s.sh"
+build_file: "grpc/tools/internal_ci/linux/psm-security.sh"
```

Comment:
@yashykt @lidizheng We're renaming "xDS K8S" job to "PSM Security", to reflect its actual purpose. Would you be ok with changing job name to `psm-security` so we're consistent across the repos (ref https://github.com/grpc/grpc-java/pull/8695), or would you insist on snake_case here? Something like `grpc_psm_security`?

Reply:
Looks like golang accepted `psm-security` too: https://github.com/grpc/grpc-go/pull/4979
cc @easwars

Reply:
I changed Kokoro job for 3 main repos. I found most of them are using snake_case. Only this job is using hyphen to concatenate words. E.g., see the Java Kokoro folder: 

https://github.com/grpc/grpc-java/tree/master/buildscripts/kokoro

Reply:
Yea, it's a bit of a mess. Some of them use dash: `android-interop.cfg`, `gae-interop.cfg`, but most are snake_case. IRRC I asked @ejona86 about a year ago whether I should do `xds-k8s` or `xds_k8s`, and we've decided on the former.

Reply:
I also prefer `xds-k8s`. In programming languages we are forced to use `_` because  `-` is an operator so can't be part of an identifier. But there simply is no reason to use `_` in script/file names (it requires the Shift key) when `-` works.